### PR TITLE
Operator code bootstrap

### PR DESCRIPTION
**Pull request by jotak**
Most of the code was generated with the operator-sdk CLI (first commit).

Next commits show:
- some auto-generated changes while generating a dummy CRD, building and deploying
- some manual changes (podman support, makefile update, readme, fixing long names error)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.16 as builder
```
Comment (mariomac) [inline Dockerfile:2]: Nit: I'd use always latest (1.17). We can do it in future commits.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	cachev1alpha1 "github.com/jotak/network-observability-operator/api/v1alpha1"
+	"github.com/jotak/network-observability-operator/controllers"
```
Comment (mariomac) [inline main.go:35]:
```suggestion
	cachev1alpha1 "github.com/netobserv/network-observability-operator/api/v1alpha1"
	"github.com/netobserv/network-observability-operator/controllers"
```

File: PROJECT [hunk 3]
```diff
@@ -0,0 +1,19 @@
+domain: netobserv.io
+layout:
+- go.kubebuilder.io/v3
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+projectName: network-observability-operator
+repo: github.com/jotak/network-observability-operator
```
Comment (mariomac) [inline PROJECT:8]:
```suggestion
repo: github.com/netobserv/network-observability-operator
```

File: controllers/suite_test.go [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+	. "github.com/onsi/gomega"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	cachev1alpha1 "github.com/jotak/network-observability-operator/api/v1alpha1"
```
Comment (mariomac) [inline controllers/suite_test.go:33]:
```suggestion
	cachev1alpha1 "github.com/netobserv/network-observability-operator/api/v1alpha1"
```

**Review by mariomac — CHANGES_REQUESTED** [review]
Minor changes with respect to the github organization (I guess it's better to change them now)

File: go.mod [hunk 5]
```diff
@@ -0,0 +1,11 @@
+module github.com/jotak/network-observability-operator
```
[the remark this answers is not in the document]
Comment (jotak) [reply go.mod:1]: good catch! thanks

**Review by mariomac — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (jotak) [reply Dockerfile:2]: ok we can do that in next PRs

**Pull request merged**